### PR TITLE
Tests all clearhaus errors.

### PR DESCRIPTION
**Pull request by ishaimika**
Creates invalid authorizations to trigger diffrent kinds of [Clearhaus transaction status codes.](https://docs.gateway.clearhaus.com/#api-reference)

### CODE REVIEW

**Comment by ishaimika** [comment]
Still no unit tests.

File: Test/3dproblem.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+			currency: "SEK",
+			card: {
+				pan: "420000403000000",
+				expires: [ 2, 22 ],
+				csc: "987",
+			},
+		}
+		const token = connection && await Authorization.create(connection, creatable)
+		return gracely.client.malformedContent.is(token) &&
+			token.content.property == "card.pan" &&
```
Comment (fredrikbryntesson) [inline Test/3dproblem.ts:23]: I dont understand the name 3dproblem, and it should be 3Dproblem.

File: Test/addauthentication.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+				expires: [ 2, 22 ],
+				csc: "987",
+			},
+		}
+		const token = connection && await Authorization.create(connection, creatable)
+		return gracely.client.malformedContent.is(token) &&
+			token.content.property == "card.pan" &&
+			token.content.description == "Invalid card number."
+	}
+})
```
Comment (fredrikbryntesson) [inline Test/addauthentication.ts:26]: addAuthentication is better but it should follow a pattern according to the two previous.

File: Test/blocked.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+				expires: [ 2, 22 ],
+				csc: "987",
+			},
+		}
+		const token = connection && await Authorization.create(connection, creatable)
+		return gracely.client.malformedContent.is(token) &&
+			token.content.property == "card.pan" &&
+			token.content.description == "Invalid card number."
+	}
+})
```
Comment (fredrikbryntesson) [inline Test/blocked.ts:26]: These files seems to ave the same content?

**Comment by ishaimika** [comment]
> I dont understand this PR, why are we adding the same files with different names? How does this work? I dont see how the names are describing the test.

Changed PR comment to describe it better:
"Creates invalid authorizations to trigger diffrent kinds of [Clearhaus transaction status codes.](https://docs.gateway.clearhaus.com/#api-reference)"

Also added a description on all tests to correspond with Clearhaus transaction status codes.

**Pull request merged**